presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -772,4 +772,24 @@ public void testWidthBucketArray()
         // this is a case that we can't catch because we are using binary search to bisect the bins array
         assertFunction("width_bucket(1.5, array[1.0, 2.3, 2.0])", BIGINT, 1L);
     }
+
+    @Test
+    public void testCosineSimilarity()
+    {
+//        assertFunction("cosine_similarity( map(array ['a', 'b'], array [1.0, 2.0]), map(array ['c', 'b'], array [1.0, 3.0]) )",
```

Comment:
These look useful. Why commented out?

We don't add commented out code. Either remove or uncomment